components/button/index.css

```diff
@@ -306,6 +306,10 @@ a.spectrum-Button {
 /* Windows High Contrast Mode */
 @media (forced-colors: active) {
```

Comment:
While we're cleaning up these high contrast styles, could you bring over some of the changes I made in the spectrum 2 migration here? The use of the `Highlight` background should be paired with `HighlightText`: https://github.com/adobe/spectrum-css/pull/2600/commits/597d33fa650566445797d4f50fcedb29856cfde4

Reply:
Let me know if this lines up with what you're expecting https://github.com/adobe/spectrum-css/pull/3078/commits/853ddac2d64748b9dadadc5253614ac63b61e0ff#diff-97b5a7e747486aca74b3076354c28a00ef69dab3c91dc0806d926078198571a2R335 

Thanks for taking a look! 🙂

Reply:
Thanks! Could you keep the `&.spectrum-Button--accent.spectrum-Button--fill` selector the same for now? Removing the fill class hasn't been done yet in S1, so keeping that the same should avoid needing changes on the SWC side.

Reply:
Yep! Just reverted that.